apps/prairielearn/src/middlewares/authzCourseOrInstance.sql

```diff
@@ -42,17 +41,10 @@ WHERE
       permissions_course_instance ->> 'course_instance_role'
     )::enum_course_instance_role > 'None'
     OR (
-      permissions_course_instance ->> 'has_student_access_with_enrollment'
+      permissions_course_instance ->> 'has_student_access'
```

Comment:
This changes when a row is returned from this query; please check that I'm handling things in JavaScript correctly!